pkg/api/meta/restmapper.go

```diff
@@ -503,6 +503,49 @@ func (m *DefaultRESTMapper) RESTMapping(gk unversioned.GroupKind, versions ...st
 	return retVal, nil
 }
 
+// RESTMappings returns the RESTMappings for the provided group kind in a rough internal preferred order. If no
+// kind is found it will return a NoResourceMatchError.
+func (m *DefaultRESTMapper) RESTMappings(gk unversioned.GroupKind) ([]*RESTMapping, error) {
```

Comment:
> I'd rather not destabilize this method right now.  I agree it could be
> more tightly factored, but on the other hand it's been working for a
> very long time.

:(  Issue.